### PR TITLE
Add napari-cursor-tracker recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

---

@jaimergp, do you agree with being added as maintainer?

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/napari-cursor-tracker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+test:
+  imports:
+    - napari_cursor_tracker
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+    - pyqt
```
Comment (xhochy) [inline recipes/napari-cursor-tracker/meta.yaml:39]: Same qt issue again. I have probably overlooked this in previous reviews but we should fix this. In contrast to pip we have the means with conda and thus should always provide the user with a fully functional setup.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/napari-cursor-tracker/meta.yaml (hunk 1 above)
Comment (goanpeca) [reply recipes/napari-cursor-tracker/meta.yaml:39]: @xhochy again, same comment from previous recipe (and this is something that has been  decided for napari). Napari by default does not provide a binding also on pip, so a user needs to install napari and then a binding, pyside or pyqt, hence why all these plugins will need to include a binding so the imports or tests pass.

This is not something we need to fix. Is a packaging decision. See https://napari.org/stable/tutorials/fundamentals/installation.html

Comment (goanpeca) [reply recipes/napari-cursor-tracker/meta.yaml:39]: The variant option already discussed and voted against.

See:
- https://github.com/conda-forge/napari-feedstock/pull/48
- https://github.com/conda-forge/napari-feedstock/issues/47

The napari team is ok with having this as is for both conda and pip.

Comment (xhochy) [reply recipes/napari-cursor-tracker/meta.yaml:39]: Fine with me then.

**Review by xhochy — APPROVED** [review]

**Pull request merged**